internal/backends/winit/wasm_input_helper.rs

```diff
@@ -112,18 +105,17 @@ impl WasmInputHelper {
             if let (Some(window_adapter), Some(data)) = (win.upgrade(), e.data()) {
                 if !e.is_composing() && e.input_type() != "insertCompositionText" {
                     if !shared_state2.borrow_mut().has_key_down {
-                        let window_inner = WindowInner::from_pub(window_adapter.window());
                         let text = SharedString::from(data.as_str());
```

Comment:
```suggestion
                        let text = data.as_str();
```

And probably `text` is not even needed and one can just call `data.chars()`

Reply:
Good catch!